.github/workflows/ci.yml

```diff
@@ -42,17 +42,16 @@ jobs:
       matrix:
         include:
           # Linux, gcc
-          - { compiler: gcc-5, cxxstd: '14',       boostBranch: boost-1.65.0, os: ubuntu-22.04, container: 'ubuntu:18.04' }
```

Comment:
Only container jobs are affected. There is a workaround to still use the v3 actions which would work but it isn't sure for how long. So I guess we can drop every OS not natively available.

With ubuntu-20.04 we could test GCC 7 so I'd use that as the lowest one. 12 is quite new

Reply:
It seems the workaround doesn’t work anymore, see the `main` build failures
```
/__e/node20/bin/node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /__e/node20/bin/node)
```
I.e. https://github.com/actions/checkout/issues/1590

Reply:
> With ubuntu-20.04 we could test GCC 7 so I'd use that as the lowest one. 12 is quite new

Sure, I’ll do that!

Reply:
It seems to be failing with gcc 7: https://github.com/mat007/turtle/actions/runs/11085108006/job/30801140786?pr=124

Reply:
Latest release for gcc 7 was 7.5 on November 14, 2019 see https://gcc.gnu.org/releases.html
That looks pretty old to me 🤔

Reply:
GCC 7 just doesn't support C++20 which is that failure. And yes it is 5 yrs old, but for some this is "only 5 yrs". So similar how previously we had GCC 5 with C++14 as the oldest one and we can test GCC 7  easily IMO we should